labs/Certified_Anonymous_Identities-Fabric.md

```diff
@@ -0,0 +1,14 @@
+# Certified Anonymous Identities in Hyperledger Fabric
+
+
+# Scope of Lab
+The scope of this lab is to achieve an implementation of Hyperledger Fabric where the user may decide to remain anonymous thanks to a cryptographic protocol but can give a certified proof sharing some kind of information (for example age, eye color etc.) given the proof that information is true thanks to a special consesus algorithm.
```

Comment:
Typo in the word `consensus` at the end.

Reply:
@tkuhrt thanks